gateway-getting-started-guide-turris-omnia.md

```diff
@@ -0,0 +1,79 @@
+# Getting Started
+
+## WebThings Gateway for Turris Omnia
+
+**Warning:** WebThings Gateway for Turris Omnia builds are currently highly experimental. So far the WebThings Gateway web interface only provides very basic router settings and it is not suitable for replacing your existing router software. We do not recommend using these builds for your primary home router.
+
+<img src="/images/turris_omnia.png" width="400">
+
+[WebThings Gateway](https://iot.mozilla.org/gateway/) is a software distribution for smart home gateways which allows users to directy monitor and control their smart home over the web, without a middleman.
+
+### What you will need
+
+1. **A [Turris Omnia](https://www.turris.cz/en/omnia/)** router
+2. **A USB stick** (at least 256MB)
+3. **USB dongles** (see the list of [compatible adapters](https://github.com/mozilla-iot/wiki/wiki/Supported-Hardware#adapters))
+
+**Note:** The Turris Omnia comes with Wi-Fi built in. The USB dongles are needed if you want to support other smart home protocols like Bluetooth, Zigbee and Z-Wave.
```

Comment:
Maybe omit the Bluetooth bit, since we don't have any working Bluetooth add-ons for OpenWrt right now.